backend/src/layer/data/flickr_api/flickr_api_handler.rs

```diff
@@ -18,22 +18,23 @@ pub struct FlickrApiHandler {
 
 lazy_static! {
     static ref LONG_URL_REGEX: Regex =
-        Regex::new(r"(https://www.flickr.com/photos/)(\w+)/(\d+)([/]{0,1})")
+        Regex::new(r"(https://www.flickr.com/photos/)([\d\w@]+)/(\d+)([/]{0,1})")
```

Comment:
regex `?` for 0 or 1
you also only need to put paranteses "()" around blocks if you want to group then to extract their content later

Reply:
https://docs.rs/regex/latest/regex/struct.Captures.html
That is something new to me... but very helpful. This will be implemented in the next push.